packages/yourloops/lib/utils.ts

```diff
@@ -133,7 +134,7 @@ export function fixYLP878Settings(settings: Settings | undefined | null): Settin
       }
     }
   }
-  let bgUnit = _.get(settings, 'bg', settings.units?.bg ?? Units.gram) as Units
+  let bgUnit = _.get(settings, 'bg', settings.units?.bg ?? Units.gram)
```

Comment:
Could be simplified and also allows to remove lodash (you can remove the `_.isNil` just above)
```suggestion
  let bgUnit = settings.units?.bg ?? Units.gram
```